web/src/lib/i18n/locales/ja.json

```
@@ -1,9 +1,21 @@
 {
+	"layout": {
+		"header": {
+			"home": "ホーム",
+			"search": "検索",
+			"global": "日本語リレー",
```

nostter の Global(?) TL はリレー指定ではなくトレンドユーザーまたは日本語ユーザーのリストなので「リレー」の文言は消したいです。
```suggestion
			"global": "日本語",
```

---

en の方は Trend って表示するのも変なので Global のままでいいかも？